### ISSUE TITLE
i cant publish my repository from my github desktop

### ISSUE THREAD

**Issue by Ade0211** [desktop/desktop#10418]
### Describe the bug
i was able to clone my master repository on my github desktop but when i created a new branch, i could not publish it as it was giving an authentication error

my github desktop is Version 2.5.3

my operating system is interl core i5 10th GEN

Authentication failed. Some common reasons include:

- You are not logged in to your account: see File > Options.
- You may need to log out and log back in to refresh your token.
- You do not have permission to access this repository.
- The repository is archived on GitHub. Check the repository settings to confirm you are still permitted to push commits.
- If you use SSH authentication, check that your key is added to the ssh-agent and associated with your account.

### Logs
```
2020-08-18T09:56:26.996Z - info: [main] Received possible protocol arguments: 2
2020-08-18T09:56:26.997Z - error: [main] Malformed launch arguments received: C:\Users\ige_a\AppData\Local\GitHubDesktop\app-2.5.3\GitHubDesktop.exe,--squirrel-firstrun
2020-08-18T09:56:29.221Z - info: [ui] [AppStore] loading 0 repositories from store
2020-08-18T09:56:29.244Z - info: [ui] [Welcome] no sign in step found. ignoring...
2020-08-18T09:56:29.247Z - info: [ui] [Welcome] no sign in step found. ignoring...
2020-08-18T09:56:29.728Z - info: [ui] launching: 2.5.3 (Windows 10.0.18363)
2020-08-18T09:56:29.728Z - info: [ui] execPath: 'C:\Users\ige_a\AppData\Local\GitHubDesktop\app-2.5.3\GitHubDesktop.exe'
2020-08-18T09:56:29.731Z - info: [ui] Opt in reported.
2020-08-18T09:56:29.731Z - info: [ui] [Welcome] no sign in step found. ignoring...
2020-08-18T09:56:29.754Z - info: [ui] [Welcome] no sign in step found. ignoring...
2020-08-18T09:56:30.763Z - info: [ui] Executing installGlobalLFSFilter: git lfs install --skip-repo (took 1.326s)
2020-08-18T09:56:36.061Z - info: [ui] [Welcome] advancing to step: SignInToDotComWithBrowser
2020-08-18T09:56:36.065Z - info: [ui] [SignInStore] initializing OAuth flow
2020-08-18T09:56:36.068Z - info: [main] opening in browser: https://github.com/login/oauth/authorize?client_id=de0e3c7e9973e1c4dd77&scope=repo%20user%20workflow&state=b54eddaf-2c5c-4521-85e9-4631b61fedb0
2020-08-18T09:56:36.069Z - info: [ui] [Welcome] current sign in state not found. ignoring...
2020-08-18T09:56:36.088Z - info: [ui] [Welcome] kind Authentication is the same as Authentication. ignoring...
2020-08-18T09:56:36.893Z - info: [ui] [Welcome] kind Authentication is the same as Authentication. ignoring...
2020-08-18T09:56:43.780Z - info: [main] Received possible protocol arguments: 5
2020-08-18T09:56:43.780Z - info: [main] Processing protocol url
2020-08-18T09:56:43.789Z - info: [ui] [Dispatcher] requesting authenticated user
2020-08-18T09:56:43.794Z - info: [ui] [Welcome] kind Authentication is the same as Authentication. ignoring...
2020-08-18T09:56:43.882Z - info: [main] Received possible protocol arguments: 3
2020-08-18T09:56:43.884Z - info: [main] Processing protocol url
2020-08-18T09:56:44.578Z - info: [ui] [SignInStore] account resolved
2020-08-18T09:56:44.578Z - info: [ui] [AppStore] adding account Ade0211 (Ade) to store
2020-08-18T09:56:44.591Z - info: [ui] [Welcome] advancing to step: ConfigureGit
2020-08-18T09:56:44.639Z - info: [ui] [Welcome] no sign in step found. ignoring...
2020-08-18T09:56:48.922Z - info: [ui] [Welcome] no sign in step found. ignoring...
2020-08-18T09:56:54.479Z - info: [ui] [Welcome] advancing to step: UsageOptOut
2020-08-18T09:56:57.608Z - info: [ui] Opt in reported.
2020-08-18T09:56:57.624Z - info: [ui] [Welcome] no sign in step found. ignoring...
2020-08-18T09:58:28.943Z - info: [main] Received possible protocol arguments: 5
2020-08-18T09:58:28.943Z - info: [main] Processing protocol url
2020-08-18T09:58:29.042Z - info: [main] Received possible protocol arguments: 3
2020-08-18T09:58:29.046Z - info: [main] Processing protocol url
2020-08-18T09:58:56.297Z - info: [ui] [AppStore.getAccountForRemoteURL] account found for remote: https://github.com/CodeYourFuture/JavaScript-Core-3-Homework.git - Ade0211 (has token)
2020-08-18T09:59:02.025Z - info: [ui] Executing clone: git -c credential.helper= -c protocol.version=2 clone --recursive --progress -- https://github.com/CodeYourFuture/JavaScript-Core-3-Homework.git C:\Users\ige_a\Desktop\CYF\JavaScript-Core-3-Homework (took 5.508s)
2020-08-18T09:59:02.121Z - info: [ui] [AppStore] adding repository at C:\Users\ige_a\Desktop\CYF\JavaScript-Core-3-Homework to store
2020-08-18T10:04:14.271Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.978s)
2020-08-18T10:35:13.788Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 2.081s)
2020-08-18T11:06:13.495Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 2.060s)
2020-08-18T11:37:13.724Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 2.431s)
2020-08-18T12:08:13.842Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.976s)
2020-08-18T12:17:48.082Z - info: [ui] [Timing] Action 'create branch' for 'codeyourfuture/JavaScript-Core-3-Homework' took 0.208s
2020-08-18T12:17:59.222Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: JavaScript-Core-3-Homework - Ade0211 (has token)
2020-08-18T12:18:00.011Z - info: [ui] [Timing] Action 'stash and checkout' for 'codeyourfuture/JavaScript-Core-3-Homework' took 0.794s
2020-08-18T12:18:51.275Z - info: [ui] [Timing] Action 'create commit' for 'codeyourfuture/JavaScript-Core-3-Homework' took 1.515s
2020-08-18T12:18:53.576Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: JavaScript-Core-3-Homework - Ade0211 (has token)
2020-08-18T12:18:56.499Z - info: [ui] Executing push: git -c credential.helper= -c protocol.version=2 push origin javascript/Adebola/core-3 --set-upstream --progress (took 2.814s)
2020-08-18T12:19:13.133Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: JavaScript-Core-3-Homework - Ade0211 (has token)
2020-08-18T12:19:20.078Z - info: [ui] Executing push: git -c credential.helper= -c protocol.version=2 push origin javascript/Adebola/core-3 --set-upstream --progress (took 6.828s)
2020-08-18T12:19:21.748Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.550s)
2020-08-18T12:20:37.737Z - info: [main] Received possible protocol arguments: 5
2020-08-18T12:20:37.737Z - info: [main] Processing protocol url
2020-08-18T12:20:37.822Z - info: [main] Received possible protocol arguments: 3
2020-08-18T12:20:37.824Z - info: [main] Processing protocol url
2020-08-18T12:21:11.327Z - warn: [ui] Open Repository from URL failed, did not find or clone repository: https://github.com/Ade0211/tv-show-dom-project - payload: {"branch":null,"filepath":null,"name":"open-repository-from-url","pr":null,"url":"https://github.com/Ade0211/tv-show-dom-project"}
2020-08-18T12:22:46.547Z - info: [ui] [AppStore] loading 6 repositories from store
2020-08-18T12:22:46.548Z - info: [ui] [AppStore] found account: Ade0211 (Ade)
2020-08-18T12:22:47.059Z - info: [ui] launching: 2.5.3 (Windows 10.0.18363)
2020-08-18T12:22:47.059Z - info: [ui] execPath: 'C:\Program Files\WindowsApps\6530varisteGalois.GitHubDesktop_0.0.1.0_x64__gcbhn7m1f6q1a\VFS\Local AppData\GitHubDesktop\app-2.5.3\GitHubDesktop.exe'
2020-08-18T12:22:47.572Z - info: [ui] Stats reported.
2020-08-18T12:22:54.415Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.367s)
2020-08-18T12:22:57.025Z - info: [ui] calling captureCommandOutput(where git)
2020-08-18T12:24:52.147Z - info: [ui] Background fetch for 6 repositories took 5.521sec
2020-08-18T12:39:58.212Z - info: [ui] Background fetch for 6 repositories took 3.905sec
2020-08-18T12:53:38.419Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.338s)
2020-08-18T12:54:57.949Z - info: [ui] Background fetch for 6 repositories took 3.651sec
2020-08-18T14:17:31.366Z - warn: [ui] fetchRepository: an error occurred for 'Ade0211/Guess-The-Number-Game'
TypeError: Failed to fetch
2020-08-18T14:17:37.101Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 8.526s)
2020-08-18T14:17:37.778Z - info: [ui] Background fetch for 6 repositories took 9.928sec
2020-08-18T14:24:57.785Z - info: [ui] Background fetch for 6 repositories took 3.506sec
2020-08-18T14:39:58.425Z - info: [ui] Background fetch for 6 repositories took 4.146sec
2020-08-18T14:48:18.948Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.308s)
2020-08-18T14:54:57.838Z - info: [ui] Background fetch for 6 repositories took 3.560sec
2020-08-18T15:09:57.797Z - info: [ui] Background fetch for 6 repositories took 3.507sec
2020-08-18T15:19:00.999Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.439s)
2020-08-18T15:24:57.901Z - info: [ui] Background fetch for 6 repositories took 3.609sec
2020-08-18T15:39:58.022Z - info: [ui] Background fetch for 6 repositories took 3.743sec
2020-08-18T15:49:42.803Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.297s)
2020-08-18T15:54:58.604Z - info: [ui] Background fetch for 6 repositories took 4.325sec
2020-08-18T16:09:58.620Z - info: [ui] Background fetch for 6 repositories took 4.340sec
2020-08-18T16:20:24.824Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.529s)
2020-08-18T16:22:49.344Z - info: [ui] [BranchPruner] No branches to prune.
2020-08-18T16:24:58.427Z - info: [ui] Background fetch for 6 repositories took 4.147sec
2020-08-18T16:39:59.028Z - info: [ui] Background fetch for 6 repositories took 4.748sec
2020-08-18T16:51:08.068Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.722s)
2020-08-18T16:54:59.202Z - info: [ui] Background fetch for 6 repositories took 4.923sec
2020-08-18T17:09:59.607Z - info: [ui] Background fetch for 6 repositories took 5.326sec
2020-08-18T17:21:50.539Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.304s)
2020-08-18T17:24:57.902Z - info: [ui] Background fetch for 6 repositories took 3.622sec
2020-08-18T17:39:58.141Z - info: [ui] Background fetch for 6 repositories took 3.861sec
2020-08-18T17:52:32.293Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.314s)
2020-08-18T17:54:57.749Z - info: [ui] Background fetch for 6 repositories took 3.469sec
2020-08-18T18:09:57.848Z - info: [ui] Background fetch for 6 repositories took 3.567sec
2020-08-18T18:23:14.112Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.284s)
2020-08-18T18:24:57.823Z - info: [ui] Background fetch for 6 repositories took 3.545sec
2020-08-18T18:39:59.239Z - info: [ui] Background fetch for 6 repositories took 4.960sec
2020-08-18T18:53:56.512Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.680s)
2020-08-18T18:54:59.441Z - info: [ui] Background fetch for 6 repositories took 5.159sec
2020-08-18T19:09:59.147Z - info: [ui] Background fetch for 6 repositories took 4.865sec
2020-08-18T19:24:39.704Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.484s)
2020-08-18T19:24:58.554Z - info: [ui] Background fetch for 6 repositories took 4.274sec
2020-08-18T19:39:59.158Z - info: [ui] Background fetch for 6 repositories took 4.878sec
2020-08-18T19:54:58.589Z - info: [ui] Background fetch for 6 repositories took 4.309sec
2020-08-18T19:55:22.117Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.483s)
2020-08-18T20:10:00.353Z - info: [ui] Background fetch for 6 repositories took 6.074sec
2020-08-18T20:22:49.081Z - info: [ui] [BranchPruner] Last prune took place in 20 hours - skipping
2020-08-18T20:24:59.279Z - info: [ui] Background fetch for 6 repositories took 5.000sec
2020-08-18T20:26:04.715Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.495s)
2020-08-18T20:39:58.012Z - info: [ui] Background fetch for 6 repositories took 3.733sec
2020-08-18T20:54:58.059Z - info: [ui] Background fetch for 6 repositories took 3.779sec
2020-08-18T20:56:46.841Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.295s)
2020-08-18T21:09:58.066Z - info: [ui] Background fetch for 6 repositories took 3.788sec
2020-08-18T21:24:58.563Z - info: [ui] Background fetch for 6 repositories took 4.284sec
2020-08-18T21:27:28.682Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.309s)
2020-08-18T21:39:59.173Z - info: [ui] Background fetch for 6 repositories took 4.893sec
2020-08-18T21:54:57.737Z - info: [ui] Background fetch for 6 repositories took 3.459sec
2020-08-18T21:58:10.588Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.322s)
2020-08-18T22:09:57.744Z - info: [ui] Background fetch for 6 repositories took 3.466sec
2020-08-18T22:24:58.163Z - info: [ui] Background fetch for 6 repositories took 3.884sec
2020-08-18T22:28:52.359Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.284s)
2020-08-18T22:39:58.345Z - info: [ui] Background fetch for 6 repositories took 4.086sec
2020-08-18T22:45:21.197Z - info: [ui] [Timing] Action 'create commit' for 'ade0211/tv-show-dom-project' took 1.005s
2020-08-18T22:45:23.386Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: tv-show-dom-project - Ade0211 (has token)
2020-08-18T22:45:39.172Z - info: [ui] calling captureCommandOutput(where git)
2020-08-18T22:52:19.388Z - info: [ui] [AppStore] loading 6 repositories from store
2020-08-18T22:52:19.389Z - info: [ui] [AppStore] found account: Ade0211 (Ade)
2020-08-18T22:52:19.629Z - info: [ui] [BranchPruner] Last prune took place in 18 hours - skipping
2020-08-18T22:52:19.788Z - info: [ui] launching: 2.5.3 (Windows 10.0.18363)
2020-08-18T22:52:19.789Z - info: [ui] execPath: 'C:\Program Files\WindowsApps\6530varisteGalois.GitHubDesktop_0.0.1.0_x64__gcbhn7m1f6q1a\VFS\Local AppData\GitHubDesktop\app-2.5.3\GitHubDesktop.exe'
2020-08-18T22:52:36.490Z - info: [ui] [Timing] Action 'create commit' for 'ade0211/tv-show-dom-project' took 0.756s
2020-08-18T22:52:38.606Z - info: [ui] [AppStore.withAuthenticatingUser] account found for repository: tv-show-dom-project - Ade0211 (has token)
2020-08-18T22:52:46.444Z - info: [ui] calling captureCommandOutput(where git)
2020-08-18T22:52:46.561Z - info: [ui] calling captureCommandOutput(where git)
2020-08-18T22:54:24.151Z - info: [ui] Background fetch for 6 repositories took 4.671sec
2020-08-18T23:03:04.668Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.402s)
2020-08-18T23:09:28.578Z - info: [ui] Background fetch for 6 repositories took 4.416sec
2020-08-18T23:24:27.726Z - info: [ui] Background fetch for 6 repositories took 3.574sec
2020-08-18T23:34:34.926Z - info: [ui] Executing fetch: git -c credential.helper= -c protocol.version=2 fetch --progress --prune origin (took 1.285s)
2020-08-18T23:39:27.494Z - info: [ui] Background fetch for 6 repositories took 3.341sec
2020-08-18T23:54:27.695Z - info: [ui] Background fetch for 6 repositories took 3.543sec
```

**Comment by tierninho** [comment]
@steveward Any ideas here? Thanks!

**Comment by Msadr471** [comment]
Hi,

I have the same issue.
![image](https://user-images.githubusercontent.com/49529241/91201837-31dbb300-e716-11ea-90e6-3ce6ba4b257c.png)

and I added SSH keys:
![image](https://user-images.githubusercontent.com/49529241/91201911-4ddf5480-e716-11ea-9041-b053cb8efae8.png)

so whats problem ?


and one more thing, why LOG folder is empty???
![image](https://user-images.githubusercontent.com/49529241/91202078-897a1e80-e716-11ea-9e6e-3aaf4bc11e1a.png)

**Comment by tierninho** [comment]
@Msadr471 Please open a new issue here https://github.com/desktop/desktop/issues/new?template=bug_report.md so we can address your situation independently. Thank you.

**Comment by tierninho** [comment]
@Ade0211 Did you resolve this? as you opened https://github.com/desktop/desktop/issues/10443 which says:

> i was able to open my repository on my github but when i make changes. 

Thank you.

**Comment by Ade0211** [comment]
No I didn’t resolve it. I am just saying I can open my repository on my
GitHub but when I do anything to the repository, I can’t push it and I am
stopped from making vomits.


---------- Forwarded message ---------

**Comment by tierninho** [comment]
Gonna close this too as I believe it was resolved with #10443 👍
